docs/index.md

```diff
@@ -260,6 +260,10 @@ if __name__ == "__main__":
     asyncio.run(main())
 ```
 
+!!! note
+    Both `server.run` and `server.serve` install custom signal handlers while the server runs.
+    The original signal handlers are restored and, if appropriate, invoked once the server finishes.
```

Comment:
Without a code example here, I'm not so sure how helpful this message is.

Reply:
Would you prefer a code example or removing the message? As far as I can tell, adding custom signal handlers isn't all that common, and restoring the default signal handlers doesn't make sense as an example.

Reply:
I would prefer for it to be removed, cause I believe your PR is the logical behavior.

Reply:
```suggestion
```